proposals/vprim/README.md

```diff
@@ -0,0 +1,121 @@
+Readers may agree that some of these nondiegetic primitives are distinct enough to warrant new schemas, but may reject the premise that screen independence is necessarily a fundemental property of `Gprim`s. The USD Glossary says `Gprim`s are just drawable, transformable, and boundable. The problems of how to bound, transform (and otherwise collide / intersect) screen space objects should be clarified and reconciled within the existing `Gprim` definition, rather than introduce a new type. This is not the preferred approach of this proposal which sees value having a high level classification for diegetic and nondiegetic primitives that clients can use to organize scenes and imaging pipelines around.
+
+### Disaggregate Boundability out of `Gprim`
+The proposal suggests that `Vprim`s required a relaxed definition of boundability. One could imagine that just as `visibility` was disaggregated out of `Gprim` into its own schema, boundability could disaggregated as well so that not every `Gprim` requires a traditional `extent`.
+
+We think there might be other core properties (`doublesided`) and behaviors that warrant removal from the "visualization" domain so this isn't our preferred approach.
+
+## Summary
+To preserve compatability, supportability, and interchange of existing `UsdGeom` schemas while extending USD's ability to describe nondiegetic visualizations, a new domain should be introduced. The `Vprim` domain would allow visualizations to be described in relation to both screen and other primitives and would avoid introducing schemas that might "slice" the current set of `Gprim`s.
+
+## Questions
+* As formulated `Vprims` cannot be `PointBased` which are strictly `Gprims`. Are there any cases where dispatching behavior off of the more general `PointBased` without knowing the concrete type is important?
+* When defining screen space sizes, what is the best choice of units? Is it pixels or something else?
+* How should scaling be treated for screen space objects?
+
+## Future Work
+
+### Hiding
+As currently described, visualization primitives should be strictly hidden via depth. However, one could imagine customizing hiding for these prims. Some visualizations might want to hide based on a particular reference point for the entire string of text. Some visualizations might want to express priority with respect to other objects in the scene. Hiding is another axis where the specification may eventually end up diverging from the `Gprim` space.
+
+### Breaking the Fourth Wall
+Some care needs to be taken to establish best practices and rules for visibility of screen-constrained primitives. While we generally would expect nondiegetic primitives to not participate in physically based lighting, consider screen constrained "context spheres" as reference objects. They "break the fourth wall" and capture light and reflections. It should be possible to specify this behavior even when using nondiegetic transformations.
```

Comment:
This one somewhat throws my thoughts about the proposal into disarray... it doesn't submit to the same clean "let a rasterizer handle it in a post-pass", and it also goes against the final section below, as a fully formed material specification is required to capture light and reflections...

Reply:
Look for some clarifications in a revision shortly.  My hope would be that we could establish some set of default behavior for objects that works for most visualizations and consider if additional schemas or configurations are needed to bridge the gaps.  (It's possible the set of effects that can be described using schemas without a material network ends up being too narrow.)

Reply:
It does seem like there are a few entangled concepts here:
1.) Objects whose position or size are dependent on the camera.
2.) Objects that don't participate in light transport.
3.) Possibly redundant with 1 & 2, but stylizations done in the image plane.  Objects in categories 1 and 2 can be placed in the physical scene, if not in USD then at least in the renderer, and I suppose a silhouette *could* be constructed in the 3d scene, but that wouldn't be my choice for implementation.

If we handle concerns 1 & 2 separately, having screen-constrained primitives that participate in light transport is easy.

Reply:
Agreed-- I'm working on a revision as we speak that tries to formalize some of the ideas and came to the conclusion that it's better to keep some of these concerns separate.

Reply:
using my few functional brain cells to parse "nondiegetic break in a single diegetic space";;;;; 

"nondiegetic break in a single diegetic space" 

means

 "a thing on the stage that is not part of the world represented by the stage". If that's correct, could your amendment parenthetically add the explanation for people who need more sleep?

Reply:
Thanks Nick.  I'll try to clarify that statement more in my revisions.  The original idea of this document assumed that all nondiegetic effects were nonphysical and therefore should be excluded from lighting.  Talking to Dhruv and others, more narrow use cases emerged where it makes sense to have physically based materials and geometry constrained to view orientation.

So the revision for "anchoring" will introduce no constraints on lighting which aligns with Tom's observation that a few assumptions and usages were entangled.